ui/src/components/App.tsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import LoginScreen from './LoginScreen';
 import MainScreen from './MainScreen';
-import DamlLedger from '../daml-react/DamlLedger';
+import DamlLedger from '@daml/react/DamlLedger';
```

Comment:
```suggestion
import DamlLedger from '@daml/react';
```
The `index.ts` of `@daml/react` re-exports `DamlLedger` as the default export.